app/controllers/idv/socure/errors_controller.rb

```diff
@@ -11,23 +11,19 @@ class ErrorsController < ApplicationController
 
       before_action :confirm_step_allowed
 
-      def show
-        error_code = error_code_for(handle_stored_result)
+      def show(error_code: nil)
```

Comment:
Same as above: Since this is an action, should we check the params hash to get the error_code?
```suggestion
        def show
          error_code = params[:error_code]
```